### PR TITLE
Fixed Brain tumor code

### PR DESCRIPTION
**Pull request by jaidh01**
Closes: #130 

- **Title**:  Fixed Brain Tumor Code
- **Your Name**:  Jai Dhingra
- **Idenitfy yourself**: GSSOC-extd Contributor, Hacktoberfest

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
In the brainTumorModel.py, a variable `name` was called without Initializing it.
So, I just Initialized it.

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/Contributing.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷
Screenshot (59)

https://github.com/user-attachments/assets/92af04e4-bbc1-43b8-88e8-944dd16738ea

### CODE REVIEW

File: src/apps/pages/models/HealthCareModels/BrainTumorModel/brainTumorModel.py [hunk 1]
```diff
@@ -29,24 +29,27 @@ def do_test(img):
...
-      st.error(f'Hi {st.session_state.name},\nYou are diagnosed with Meningioma.\nPlease consult a doctor.')
-      Speak(f'Hi {st.session_state.name},You are diagnosed with Meningioma. Please consult a doctor.')
-    elif res == 2:
-      st.success(f'Congrats {st.session_state.name}, You are not diagnosed with brain tumor.')
-      Speak(f'Congrats {st.session_state.name}, You are not diagnosed with brain tumor.')
-    elif res == 3:
-      st.error(f'Hi {st.session_state.name},\nYou are diagnosed with Pituitary tumor. Please consult a doctor.')
-      Speak(f'Hi {st.session_state.name},You are diagnosed with Pituitary tumor. Please consult a doctor.')
+    if "name" not in st.session_state:
+        st.session_state.name = "User"
```
Comment (Avdhesh-Varshney) [inline src/apps/pages/models/HealthCareModels/BrainTumorModel/brainTumorModel.py:33]: Remove this session state `name`. Directly use 'User' or use other comment on the result.

File: requirements.txt
Comment (Avdhesh-Varshney) [inline requirements.txt:1]: Undo these changes for this branch. Commit only necessary things only. Remove the object detection yolo files also.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (jaidh01) [reply requirements.txt:1]: Hi, by mistake I deleted the requirements.txt file from this PR.
I think this will not effect the overall structure.

Am I right?

File: src/apps/pages/models/HealthCareModels/BrainTumorModel/brainTumorModel.py (hunk 1 above)
Comment (jaidh01) [reply src/apps/pages/models/HealthCareModels/BrainTumorModel/brainTumorModel.py]: I Removed the st.session.state.name.
And now I am Directly using `User`.

File: requirements.txt
Comment (Avdhesh-Varshney) [reply requirements.txt:1]: Don't delete it

Comment (jaidh01) [reply requirements.txt:1]: Can I undo that commit?
Or I have to add it again!!

Comment (Avdhesh-Varshney) [reply requirements.txt:1]: Yes or resolve the conflicts before approving

**Comment by jaidh01** [comment]
Hi @Avdhesh-Varshney , I have done all the changes in this PR and solved all the conflicts.
Hence, this PR is now ready for review.

**Review by Avdhesh-Varshney — APPROVED** [review]

**Comment by jaidh01** [comment]
Hi @Zedoman, is it good, or do you think any changes are needed?

**Review by Zedoman — APPROVED** [review]

**Comment by jaidh01** [comment]
Hi @Zedoman @Avdhesh-Varshney, thanks for merging the PR.
Just a small request, can you please add the `level` label as well to this PR.
